react-hot-button-component/src/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom/client';
+
+const buttonStyle = {
```

Comment:
Don't use global variables to control state in React components. It will get messed up if you have a component on the page more than once. Just put this inside the `HotButton` function.